packages/java-parser/test/records/records-spec.js

```diff
@@ -25,4 +25,42 @@ describe("The Java Parser fixed bugs", () => {
     `;
     expect(() => javaParser.parse(input, "compilationUnit")).to.not.throw();
   });
+
+  it("should handle Java records with simplified constructors inside java class declaration", () => {
```

Comment:
Was the issue only in class declaration ? That was a nasty bug ^^

Reply:
I had another bug with a record defined with a construct with args